Talkable/Talkable.m

```diff
@@ -602,28 +591,6 @@ - (void)scheduleRegisterInstall:(NSTimeInterval)delay {
     [self performSelector:@selector(registerInstallIfNeeded) withObject:nil afterDelay:delay];
 }
 
-#pragma mark - [Retrieve Rewards]
-
-- (void)retrieveRewardsIfNeeded {
-    [self retrieveRewardsWithHandler:^(NSDictionary* response, NSError* error) {
```

Comment:
Do we still need `retrieveRewardsWithHandler`?

Reply:
yes, now it is main way of getting rewards that should be called manually